samples/src/test/java/org/autorefactor/refactoring/rules/samples_out/SwitchSample.java

```diff
@@ -361,4 +361,169 @@ public void doNotReplaceWithSwitchOnPrimitiveWrapper(Integer i1) {
             i = 30;
         }
     }
+
+    public void mergeCases(int i1) {
+        int i = 0;
+        switch (i1) {
+        case 0:
+        case 1:
+            i = 0;
+            break;
+        default:
+            i = 1;
+            break;
+        }
+    }
+
+    public void mergeCasesWithUnbrokenCase(int i1) {
+        int i = 0;
+        switch (i1) {
+        case 1:
+            i = 1;
+        case 0:
```

Comment:
Firstly, to be sure I understand you, what you are saying is:
Given:
```Java
case 0:
    i = 0;
    break;
case 1:
default:
    i = 1;
    break;
```

When: refactor

Then:
```Java
case 0:
    i = 0;
    break;
default:
    i = 1;
    break;
```

OK, I will do this. I only can code this in two days. I would rather leave this because I think it's less ambiguous but it's OK.